tsconfig.json

```diff
@@ -26,7 +26,7 @@
 
     /* Modules */
     "module": "commonjs",                                /* Specify what module code is generated. */
-    "rootDir": "./",                                  /* Specify the root folder within your source files. */
+    "rootDir": "./src",                                  /* Specify the root folder within your source files. */
```

Comment:
Make src your root dir. This means that the outputted dist dir won't be 'dist/src'. Will also remove package.json from your dist folder. 

Is this going to break things for people though? There might be people importing from "harproxyserver/dist/src/file.js"